src/main/java/com/starkinfra/PixClaim.java

```diff
@@ -114,10 +109,8 @@ public PixClaim(String accountCreated, String accountNumber, String accountType,
     /**
      * PixClaim object
      * <p>
-     * PixClaims intend to transfer a PixKey from one account to another.
-     * When you initialize a PixClaim, the entity will not be automatically
-     * created in the Stark Infra API. The 'create' function sends the objects
-     * to the Stark Infra API and returns the list of created objects.
+     * A Pix Claim is a request to transfer a Pix Key from an account hosted at another
+     * Pix participant to an account under your bank code. Pix Claims must always be requested by the claimer.
```

Comment:
Aqui tmb. um bom jeito pra checar isso é contar o número de vezes que `When you initialize a` aparece em cada arquivo. Pra maioria deles tem que ter 3 a não ser que não exista a função create